Drop-In/src/androidTest/java/com/braintreepayments/api/BottomSheetFragmentUITest.kt

```diff
@@ -0,0 +1,195 @@
+package com.braintreepayments.api
+
+import androidx.fragment.app.testing.FragmentScenario
+import androidx.lifecycle.Lifecycle
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.matcher.ViewMatchers.isRoot
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import com.braintreepayments.api.dropin.R
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.concurrent.CountDownLatch
+
+@RunWith(AndroidJUnit4ClassRunner::class)
+class BottomSheetFragmentUITest {
+
+    private lateinit var countDownLatch: CountDownLatch
```

Comment:
I should have asked here. What is the importance/benefit of the countDownLatch in all the tests? Is there some cool tie-in with AndroidJUnit?  

I see it on a lot of the new tests, but in some cases it doesn't seem like we use it, or at least I didn't notice it after a couple of scans of the changes here.